pkg/api/converterfromadmin.go

```diff
@@ -434,6 +434,13 @@ func mergeCertKeyPair(in *admin.Certificate, out *CertKeyPair) {
 	return
 }
 
+// TODO: this is not a great semantic - revisit if this field is ever enabled for write
+func mergeCertKeyPairChain(in *admin.CertificateChain, out *CertKeyPairChain) {
+	if in.Certs != nil {
```

Comment:
can you put this back to what was there previously - ie the append semantic?  Both are questionable in this case but given we have the append semantic everywhere else, I'd rather be standard for now

Reply:
This is more tricky. Because we prepare test structure with `populate.Walk(&cs, prepare)` which does this:
```
case *x509.Certificate:
		// use a dummy value because the zero value cannot be unmarshalled
		v.Set(reflect.ValueOf(tls.GetDummyCertificate()))
		return
	}
```
We already have value in `out` field. So we either have to "nil" it, and overwrite, or we have to make sure our tests are fine.

p.s we overwrite anywhere else in the converters. And we might need to change that in the future,

Reply:
ok ;)